firmware/user/modes/mode_flight.c

```diff
@@ -230,6 +244,31 @@ static const uint16_t IsoSphereIndices[] RODATA_ATTR = { /* 120 line segments */
         };
 
 
+/**
+ * Re-setup the main flight menu.
+ */
+void ICACHE_FLASH_ATTR flightRefreshMenu(void)
+{
+    flight->menu = initMenu(fl_title, flightMenuCb);
```

Comment:
If you call this multiple times without calling `deinitMenu()`, it will leak memory